src/ankisyncd/full_sync.py

```diff
@@ -1,45 +1,81 @@
 # -*- coding: utf-8 -*-
 
+import logging
 import os
 from sqlite3 import dbapi2 as sqlite
+import shutil
+import sys
+from webob.exc import HTTPBadRequest
 
-import anki.db
+from anki.db import DB
+from anki.collection import Collection
 
-class FullSyncManager:
-    def upload(self, col, data, session):
+logger = logging.getLogger("ankisyncd.media")
+logger.setLevel(1)
+
+class FullSyncManager(object):
```

Comment:
Do you have a reason for preferring having an explicit inheritance of (object)? In python3 it makes no difference and it wasn't used before, so I'm not sure I see the value. In any case, I don't think we should mix (having and not having) and I personally don't think it serves a great purpose.

Reply:
Just a relict from the good old Python 2 days, when inheriting from `object` did indeed make a difference.